benchmarks/varint_speed.py

```diff
@@ -67,6 +65,10 @@
 BENCH_VALUES_DEC = list(map(bytearray, BENCH_VALUES_DEC))
 
 
+def int2byte(i):
+    return bytes((i),)
+
```

Comment:
> Forced pushed, let's see if I missed anything. I'm not 100% happy with the crimes I committed in `benchmarks/variant_speed.py`, but I also didn't want to completely rewrite it to remove its reliance on the Python 2 construct of int2byte.

It happens, it's a sacrifice that sometimes must be made to keep the ball rolling 😅